cmd/zed/zed.d/all-syslog.sh

```diff
@@ -5,6 +5,18 @@
 [ -f "${ZED_ZEDLET_DIR}/zed.rc" ] && . "${ZED_ZEDLET_DIR}/zed.rc"
 . "${ZED_ZEDLET_DIR}/zed-functions.sh"
 
+if [ -n "${ZED_SYSLOG_SUBCLASS_INCLUDE}" ]; then
+    case "${ZEVENT_SUBCLASS}" in
+    ${ZED_SYSLOG_SUBCLASS_INCLUDE}) ;;
+    *) exit 0;;
+    esac
+elif [ -n "${ZED_SYSLOG_SUBCLASS_EXCLUDE}" ]; then
+    case "${ZEVENT_SUBCLASS}" in
+    ${ZED_SYSLOG_SUBCLASS_EXCLUDE}) exit 0;;
```

Comment:
[nit] I'd suggest dropping the unreachable `;;` after all the exits

Reply:
Do you want me to drop the `;;` preceeding the closing `esac`? I tend to include them to avoid errors when adding further patterns or shuffling patterns around. The other `;;` aren't statements of their own but syntactically required to separate the patterns in the case statements.

Reply:
No, I'm fine with those.  And if you prefer feel free to keep the `;;` after the `exit` since it is really simply a style issue.